test/test_retryable_reads.py

```diff
@@ -68,11 +69,25 @@ def maybe_skip_scenario(self, test):
             if name.lower() in test['description'].lower():
                 self.skipTest('PyMongo does not support %s' % (name,))
 
-        # Skip changeStream related tests on MMAPv1.
+        # Serverless does not support $out and collation.
+        if client_context.serverless:
+            for operation in test['operations']:
+                if operation['name'] == 'aggregate':
+                    for stage in operation['arguments']['pipeline']:
+                        if "$out" in stage:
+                            self.skipTest(
+                                "MongoDB Serverless does not support $out")
+                if "collation" in operation['arguments']:
+                    self.skipTest(
+                        "MongoDB Serverless does not support collations")
```

Comment:
DRIVERS-1836 updated the unified tests and DRIVERS-1851 updated the crud v1 tests. Is there any reason why we did not update the retryable reads/writes tests? Can you file a drivers ticket for it?

Reply:
I think this is only a concern for Retrayble Reads. I opened [DRIVERS-1869](https://jira.mongodb.org/browse/DRIVERS-1869).